05/app.js

```diff
@@ -10,25 +10,66 @@ const stats = {
 /* tutaj umieść swój kod */
 const paragraphs = document.querySelectorAll("p");
 for (let i = 0; i < paragraphs.length; i++) {
-paragraphs[i].addEventListener("click", function () {
-        if (paragraphs[i].dataset.id === "p1") {
+paragraphs[i].addEventListener("click", function (e) {
+        if (e.target.dataset.id === "p1") {
             stats.paragraphs.p1 = stats.paragraphs.p1 + 1;
         }
+        if (e.target.dataset.id === "p2") {
+            if(!stats.paragraphs.p2) {
+                stats.paragraphs.p2 = 0;
+            }
+            
+            stats.paragraphs.p2 = stats.paragraphs.p2 + 1;
+            
+        }
+        if (e.target.dataset.id === "p3") {
+            if(!stats.paragraphs.p3) {
+                stats.paragraphs.p3 = 0;
+            }
+            
+            stats.paragraphs.p3 = stats.paragraphs.p3 + 1;
+            
+        }
```

Comment:
ok, ale co jeśli tych paragrafów będzie 100, a potem jeszcze dodamy 200?
Będziemy to w ten sposób robić? Trzeba znaleźć bardziej uniwersalne rozwiązanie.

Taka podpowiedź:
```
const key = e.target.dataset.id;
stats.paragraphs[key] += 1;
```